awx/main/utils/common.py

```diff
@@ -887,30 +888,38 @@ def get_current_apps():
     return current_apps
 
 
-def get_custom_venv_choices(custom_paths=None):
+def get_custom_venv_choices():
     from django.conf import settings
 
-    custom_paths = custom_paths or settings.CUSTOM_VENV_PATHS
```

Comment:
I don't see how we get around the need for `CUSTOM_VENV_PATHS` which is removed here. Without that, I don't see how this command could make sense.

Reply:
that command actually doesn't return anything anyways 🤷🏻

Reply:
oh my gosh @AlanCoding I actually take that back. I'm so glad you said this because I swapped this call and the one on line 896 which is why that command doesn't return anything. I'll update 🎊